openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -230,21 +230,21 @@ def test_content_locked(self):
         lock_icon = response_content('.fa-lock')
         self.assertTrue(lock_icon, "lock icon is not present, but should be")
 
-        subsection = lock_icon.parents('.subsection-title')
+        subsection = lock_icon.parents('.subsection-text')
 
         # check that subsection-title-name is the display name
         gated_subsection_title = self.course_blocks['gated_content'].display_name
-        self.assertIn(gated_subsection_title, subsection.children('.subsection-title-name').html())
+        self.assertIn(gated_subsection_title, subsection.children('.subsection-title').html())
 
         # check that it says prerequisite required
-        self.assertIn(self.PREREQ_REQUIRED, subsection.children('.details').html())
+        self.assertIn("Prerequisite:", subsection.children('.details').html())
 
         # check that there is not a screen reader message
         self.assertFalse(subsection.children('.sr'))
 
     def test_content_unlocked(self):
         """
-        Test that a sequential/subsection with unmet prereqs correctly indicated that its content is locked
```

Comment:
This kind of copypasta (that you politely fixed) is why I live by the phrase "all comments are lies".